src/logger/loggerConfig.c

```diff
@@ -263,11 +263,17 @@ int getConnectivitySampleRateLimit()
     return sampleRateLimit;
 }
 
-// STIEG: FIX ME! This can be done with math
+/* Filter sample rates to only allow rates we support */
 int encodeSampleRate(int sampleRate)
 {
-
+    if (sampleRate > MAX_SENSOR_SAMPLE_RATE){
```

Comment:
Style fail.  Should be `if (sampleRate > MAX_SENSOR_SAMPLE_RATE) {`.  Also braces not needed for one liner :).